vuln/npm/391.json

```diff
@@ -0,0 +1,19 @@
+{
+  "id": 391,
+  "title": "Memory Exposure",
+  "author": "Feross Aboukhadijeh",
+  "module_name": "bl",
+  "created_at": "2017-04-24",
+  "updated_at": "2017-04-24",
+  "publish_date": "2016-01-19",
+  "cves": [],
+  "vulnerable_versions": "<=0.9.4 || 1.0.0",
+  "patched_versions": ">=1.0.1 || >=0.9.5 <1.0.0",
+  "slug": "bl-memory-exposure",
+  "overview": "bl.append(number) in the affected `bl` versions passes a number to Buffer constructor, appeding a chunk of unitialized memory",
```

Comment:
Not sure if we want to fix small typos in this file or if they don't really matter, but if we want to fix: `appeding` -> `appending`

Reply:
I think that fixing typos makes sense, as we often modify those files in fact with various fixes. So they are not frozen.